### PR TITLE
handle validate for choices

### PR DESCRIPTION
**Pull request by haroun**
## Summary

Log only errors for visible fields + wait for choices to trigger validate

## What are the specific steps to test this change?

*For example:*
> 1. Run the website and log in as an admin
> 2. Open a piece manager modal and select several pieces
> 3. Click the "Archive" button on the top left of the manager and confirm that it should proceed
> 4. Check that all pieces have been archived properly

## What kind of change does this PR introduce?
*(Check at least one)*

- [x] Bug fix
- [ ] New feature
- [ ] Refactor
- [ ] Documentation
- [ ] Build-related changes
- [ ] Other

## Make sure the PR fulfills these requirements:

- [x] It includes a) the existing issue ID being resolved, b) a convincing reason for adding this feature, or c) a clear description of the bug it resolves
- [ ] The changelog is updated
- [ ] Related documentation has been updated
- [ ] Related tests have been updated

If adding a new feature without an already open issue, it's best to open a **feature request issue** first and wait for approval before working on it.

**Other information:**

### CODE REVIEW

File: modules/@apostrophecms/schema/index.js [hunk 1]
```diff
@@ -656,6 +673,9 @@ module.exports = {
                 continue;
               }
             }
+            if (isParentVisible === false) {
+              continue;
+            }
```
Comment (ValJed) [inline modules/@apostrophecms/schema/index.js:678]: @boutell Would like your opinion on that:
In this file just before line `672` we add default value to hidden fields.
If this is necessary, I suppose that we should do the same thing when a parent is hidden?
WDYT?

**Review by ValJed — CHANGES_REQUESTED** [review]
Just would like @boutell opinion on a point.

**Comment by haroun** [comment]
I've created https://linear.app/apostrophecms/issue/PRO-6799/set-default-values-for-invisible-fields

**Review by ValJed — DISMISSED** [review]
LGTM

**Review by boutell — APPROVED** [review]

**Review by ValJed — APPROVED** [review]
LGTM, Might be improved in the future.

**Pull request merged**